sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -1709,6 +1718,16 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
       call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
       spval,VarName,frzrn_bucket)
 
+! time step snow (in m)
+      VarName='snow'
+      call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+      spval,VarName,snownc)
+
+! time step graupel (in m)
+      VarName='graupel'
+      call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+      spval,VarName,graupelnc)
+
```

Comment:
@WenMeng-NOAA Are snow and graupel the precipitation snow and graupel at surface for the time step or some kind of averages? Thanks

Reply:
@RuiyuSun According to netcdf header information from RRFS history file:
```
float graupel(time, grid_yt, grid_xt) 
graupel:long_name = "graupel fall at this time step" ;

float snow(time, grid_yt, grid_xt) ;
snow:long_name = "snow fall at this time step" ;
```
@EricJames-NOAA may chime in.

Reply:
@WenMeng-NOAA yes, @RuiyuSun and I looked at the code together at the WAF/NWP conference in Madison last week, and we think these variables are for the time step.

Reply:
@EricJames-NOAA Thanks for confirming.